.drone.yml

```diff
@@ -68,14 +68,30 @@ pipeline:
     secrets:
       - google_key
     source: bundle
-    target: vic-ui-builds
+    target: vic-ui-builds/
     acl:
       - 'allUsers:READER'
     cache_control: 'public,max-age=3600'
     when:
       repo: vmware/vic-ui
       event: [push]
-      branch: [master, develop, 'releases/*']
+      branch: [master, develop]
+      status: success
+
+  publish-gcs-release-builds-on-pass:
+    image: 'victest/drone-gcs:1'
+    pull: true
+    secrets:
+    - google_key
```

Comment:
The indentation here looks slightly different than above. I don't recall whether anything is sensitive to that difference.

Reply:
Done